### PR TITLE
feat(#1): bootstrap operator using operator-sdk

### PR DESCRIPTION
**Pull request by dipak-pawar**
*Changes Proposed in this PR*
- Bootstrap project with following commands
  - `operator-sdk add api --api-version=codeready.io/v1alpha1 --kind=ToolChainEnabler`
  - `operator-sdk add controller --api-version=codeready.io/v1alpha1 --kind=ToolChainEnabler`

Fixes: #1

### CODE REVIEW

File: build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM alpine:3.8
```
Comment (xcoulon) [inline build/Dockerfile]: Shouldn't we use `centos:7` as the base image?

File: deploy/crds/codeready_v1alpha1_toolchainenabler_crd.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# Todo: We may remove this in future if needed at all
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: toolchainenablers.codeready.io
+spec:
+  group: codeready.io
+  names:
+    kind: ToolChainEnabler
```
Comment (xcoulon) [inline deploy/crds/codeready_v1alpha1_toolchainenabler_crd.yaml]: I wonder if we should name it `CodeReadyToolChainEnabler`, but that would make quite a long name :/

File: build/Dockerfile (hunk 1 above)
Comment (dipak-pawar) [reply build/Dockerfile]: @xcoulon, Agreed, I'll address this when setting up cico

File: deploy/crds/codeready_v1alpha1_toolchainenabler_crd.yaml (hunk 2 above)
Comment (dipak-pawar) [reply deploy/crds/codeready_v1alpha1_toolchainenabler_crd.yaml]: if you see created CRD, you will see `toolchainenablers.codeready.io`, which already contains codeready, I don't want to repeat it again.

screenshot from 2019-01-14 14-21-12

Comment (xcoulon) [reply deploy/crds/codeready_v1alpha1_toolchainenabler_crd.yaml]: ah ok, cool then!

**Pull request merged**